docs/en/operation/Performance-Tuning.md

```diff
@@ -18,21 +18,23 @@ The following is a checklist to run through to address common problems when tuni
 1. Are all nodes working?
 
    Check that the Alluxio cluster is healthy. You can check the web user interface at
-   `http://MasterHost:19999` to see if the masters and workers are working correctly from a browser.
-   Alternatively, you can run `bin/alluxio fsadmin report` to collect similar information from the console.
-   Important metrics to verify if any nodes are out of service are the number of lost workers and
-   the last heartbeat time.
+   `http://<master hostname>:19999` to see if the masters is reachable from a browser.
```

Comment:
```suggestion
   `http://<master hostname>:19999` to see if the master is reachable from a browser.
```

Reply:
Funny thing is I definitely had a thought while writing this that "this should definite _not_ be 'masters' since the non-primary ones don't serve a UI". My brain must have disagreed when it made me write this